src/ensembl/src/content/app/entity-viewer/shared/rest/rest-data-fetchers/proteinData.ts

```diff
@@ -50,11 +52,10 @@ export const fetchProteinSummaryStats = async (
 ): Promise<ProteinStats | null> => {
   const proteinStatsUrl = `https://www.ebi.ac.uk/pdbe/graph-api/uniprot/summary_stats/${xrefId}`;
 
-  const proteinStatsData:
-    | UniProtSummaryStats
-    | undefined = await apiService.fetch(proteinStatsUrl, {
-    signal
-  });
+  const proteinStatsData: UniProtSummaryStats | undefined =
```

Comment:
I know we missed it before, but the function (line 60) returns `null` rather than undefined; so we should update our type.